packages/app/hooks/param/useBeatmakerParam.ts

```diff
@@ -0,0 +1,24 @@
+import { createParam } from "solito";
+import { useMemo } from "react";
+
+export type BeatmakerParam = {
+  id: string;
+  displayName: string;
+  avatarUrl?: string;
+};
+
+const { useParam } = createParam<BeatmakerParam>();
+
+export const useBeatmakerParam = (): BeatmakerParam | undefined => {
+  const [id] = useParam("id");
+  const [displayName] = useParam("displayName");
+  const [avatarUrl] = useParam("avatarUrl");
```

Comment:
why are we passing so many things in params?

Reply:
I need them but I don't wana make fetch requests, other option would be pass only id and fetch rest from algolia

Reply:
Wait I will try something out

Reply:
The second option is better. Otherwise, the beatmaker page won't work when we paste the URL directly.